From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Otto=20Pfl=C3=BCger?= <otto.pflueger@abscue.de>
Date: Sat, 17 May 2025 20:57:40 +0200
Subject: [PATCH 102/225] dt-bindings: watchdog: add compatible for UMS9230
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The UMS9230 SoC watchdog is compatible with the one found in the SC9860
SoC.

Signed-off-by: Otto Pflüger <otto.pflueger@abscue.de>
---
 .../devicetree/bindings/watchdog/sprd,sp9860-wdt.yaml      | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/watchdog/sprd,sp9860-wdt.yaml b/Documentation/devicetree/bindings/watchdog/sprd,sp9860-wdt.yaml
index 730d9a3a3..5a52ad06f 100644
--- a/Documentation/devicetree/bindings/watchdog/sprd,sp9860-wdt.yaml
+++ b/Documentation/devicetree/bindings/watchdog/sprd,sp9860-wdt.yaml
@@ -16,7 +16,12 @@ allOf:
 
 properties:
   compatible:
-    const: sprd,sp9860-wdt
+    oneOf:
+      - items:
+          - enum:
+              - sprd,ums9230-wdt
+          - const: sprd,sp9860-wdt
+      - const: sprd,sp9860-wdt
 
   reg:
     maxItems: 1
